ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.model.js

```diff
@@ -61,19 +63,18 @@ module.exports = Backbone.AssociatedModel.extend({
     const lat = this.get('mouseLat')
     const lon = this.get('mouseLon')
     const dms = `${mtgeo.toLat(lat)} ${mtgeo.toLon(lon)}`
-    // TODO: Move leaking defensive check knowledge to usng library (DDF-4335)
-    const mgrs =
-      lat > 84 || lat < -80
-        ? undefined
-        : converter.LLtoUSNG(lat, lon, usngPrecision)
+    const mgrs = converter.isInUPSSpace(lat)
```

Comment:
Is it possible for the user to go outside of ups and utm space? What does `converter.LLtoUSNG(lat, lon, usngPrecision)` return in this case?

Reply:
I haven't changed the logic, I simply addressed the TODO comment and used the newly available library method.